### PR TITLE
add a hard to diagnose error solution

### PR DESCRIPTION
**Pull request by punkrokk**
* document ``2020-02-06 22:04:05,202 140016844996688 INFO connectionpool [-] Starting new HTTP connection (1): 169.254.169.254`` error and reason

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,6 +287,8 @@ sqs_other:
 
 If any value exist in datastore it will be taken instead of any value in aws.yaml
 
+There have been reports of hard to troubleshoot connection errors in `st2sensorcontainer.log` such as: ``2020-02-06 22:04:05,202 140016844996688 INFO connectionpool [-] Starting new HTTP connection (1): 169.254.169.254``. It's a result of turning on the sensor without copying the `aws.yaml` over to `/opt/stackstorm/configs/aws.yaml`
```
Comment (nmaludy) [inline README.md]: Maybe this belongs in a new `Known Issues` or `Troubleshooting` section? Thoughts?

**Review by nmaludy — COMMENTED** [review]
Looks find, just a small thought about placement.

File: README.md (hunk 1 above)
Comment (punkrokk) [reply README.md]: moved

**Comment by punkrokk** [comment]
@nmaludy Moved it over, Merging

**Pull request merged**